.gitignore

```diff
@@ -6,6 +6,7 @@
 /system/apps/test41_alchemy/root
 /system/**/build
 /system/**/_temp
+/system/release/apps*

```

Comment:
`system/build.sh` generates a `system/release/apps*` dir for each `system/apps*`

Reply:
I like the idea of putting the executables in the folder with the same name as source codes, but there is a catch. The system/release_image script expects all the freshly build binaries to be in release/bin folder. Will this script work correctly after your changes?

Reply:
Yes, the elf files are also added in `release/bin/`. You can check [`copyApp()` @ line 121 in `system/build.sh`](https://github.com/gabonator/LA104/blob/5f4972044c30605e8ad70bb45f4badf6446ddf40/system/build.sh#L121)